caniusepython3/overrides.json

```diff
@@ -11,6 +11,7 @@
     "functools32": "https://pypi.python.org/pypi/functools32",
     "geoip": "https://pypi.python.org/pypi/GeoIP/",
     "grab": "https://pypi.python.org/pypi/grab",
+    "gym": "https://github.com/openai/gym",
```

Comment:
Can you update this to https://github.com/openai/gym/blob/master/tox.ini so it points to the proof that the library supports Python 3?

Reply:
updated